SectionedSurface-Aplitop-1/SectionedSurface-Aplitop-1.IFC

```diff
@@ -0,0 +1,93 @@
+#19=IFCGEOMETRICREPRESENTATIONCONTEXT('Body','Model',3,$,#18,$);
+#20=IFCDIRECTION((1.,0.,0.));
+#21=IFCDIRECTION((0.,0.,1.));
+#22=IFCCARTESIANPOINT((0.,0.,0.));
+#23=IFCAXIS2PLACEMENT3D(#22,#21,#20);
+#24=IFCLOCALPLACEMENT($,#23);
+#25=IFCSITE('2zHKom8bz2oQNeL25oFkgh',#9,'Site',$,$,#24,$,$,.ELEMENT.,$,$,$,$,#2);
+#26=IFCPROJECT('0VGYxaGHb3APEG3MpCN2LN',#9,'Project','',$,'',$,$,#14);
+#27=IFCRELAGGREGATES('3s8mCI$QTDxwH4srCKLj04',#9,$,$,#26,(#25));
+#28=IFCROAD('2VBIkfjsriouJ$gISkmggg',#9,'Road','','ObjectType',#24,#272,'LongName',.COMPLEX.,.USERDEFINED.);
+#29=IFCRELAGGREGATES('3Z1$Qqe6XEcQRCqJoCdqUM',#9,$,$,#25,(#28));
+#30=IFCALIGNMENT('1hmNMQuc53NAma6AFvRPA6',$,'Eje1 (1)',$,$,#24,#49,$);
+#31=IFCALIGNMENTHORIZONTAL('2LpapCuJP0I8eaolX4R5iL',$,'Eje1 (1)',$,$,#24,$,0.);
+#32=IFCRELNESTS('2q4l6QB$b5o8KpfezYsjnf',#9,$,$,#30,(#31,#51));
+#33=IFCALIGNMENTSEGMENT('36ZpVDkF51Xh2KpEiWEhlp',$,$,$,$,#24,$,#35);
+#34=IFCCARTESIANPOINT((286.47559897,239.79766567));
+#35=IFCALIGNMENTHORIZONTALSEGMENT($,$,#34,1.50346536986339,0.,0.,107.15812752,$,.LINE.);
+#36=IFCDIRECTION((0.999655739909883,0.0262374096515884));
+#37=IFCAXIS2PLACEMENT2D(#34,#36);
+#40=IFCCARTESIANPOINT((0.,0.));
+#41=IFCDIRECTION((1.,0.));
+#42=IFCVECTOR(#41,1.);
+#43=IFCLINE(#40,#42);
+#44=IFCCURVESEGMENT(.CONTINUOUS.,#37,IFCNONNEGATIVELENGTHMEASURE(0.),IFCNONNEGATIVELENGTHMEASURE(107.15812752),#43);
+#45=IFCRELNESTS('1qVzEdro5CFf5K20Wsg4rv',#9,$,$,#31,(#33));
+#46=IFCCOMPOSITECURVE((#44),.F.);
+#47=IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Axis','Model',*,*,*,*,#19,$,.GRAPH_VIEW.,$);
+#48=IFCSHAPEREPRESENTATION(#47,'Axis','Curve2D',(#46));
+#49=IFCPRODUCTDEFINITIONSHAPE($,$,(#48));
+#50=IFCRELCONTAINEDINSPATIALSTRUCTURE('23N93zFe58jQ8A06YyvHJN',$,$,$,(#30),#25);
+#51=IFCALIGNMENTVERTICAL('3KIanMfxz9EvusDgX$ukNj',#9,'Eje1 (1)',$,$,#24,#67);
+#52=IFCALIGNMENTSEGMENT('3$lXDoL_910hvVFYFV6PPk',$,$,$,$,#24,$,#53);
+#53=IFCALIGNMENTVERTICALSEGMENT($,$,0.,107.158,45.315043,0.0401646447302114,0.0401646447302114,$,.CONSTANTGRADIENT.);
+#54=IFCCARTESIANPOINT((0.,45.315043));
+#55=IFCDIRECTION((0.999194375250509,0.0401322870983622));
+#56=IFCAXIS2PLACEMENT2D(#54,#55);
+#59=IFCCARTESIANPOINT((0.,0.));
+#60=IFCDIRECTION((1.,0.));
+#61=IFCVECTOR(#60,1.);
+#62=IFCLINE(#59,#61);
+#63=IFCCURVESEGMENT(.CONTINUOUS.,#56,IFCNONNEGATIVELENGTHMEASURE(0.),IFCNONNEGATIVELENGTHMEASURE(107.244398741871),#62);
+#64=IFCRELNESTS('3rAxwoD390Z8xxMdCl9q5h',#9,$,$,#51,(#52));
+#65=IFCGRADIENTCURVE((#63),.F.,#46,$);
+#66=IFCSHAPEREPRESENTATION(#47,'Axis','Curve3D',(#65));
+#67=IFCPRODUCTDEFINITIONSHAPE($,$,(#66));
+
+
+#272=IFCPRODUCTDEFINITIONSHAPE($,$,(#273));
+#273=IFCSHAPEREPRESENTATION(#19,'Profiles','',(#274));
+#274=IFCSECTIONEDSURFACE(#65,(#275,#277,#279,#281,#283),(#276,#278,#280,#282,#284),.F.);
+
+#275=IFCOPENCROSSPROFILEDEF(.CURVE.,$,.T.,(0.031,3.5,3.5,0.056),(-0.667,0.,0.,-0.667),('119','30','0','30','139'));
+#276=IFCPOINTBYDISTANCEEXPRESSION(IFCLENGTHMEASURE(0.),IFCLENGTHMEASURE(3.531),0.,0.,#65);
```

Comment:
From looking at the xml data, it would seem that there should be a negative vertical offset needed in this case. Also, i am not sure where the value 3.531 comes from. In the xml at station 0 we have -2.736 and at station 20 we have -3.778.

Reply:
Value 3.351 is a left displacement of the OpenCrossProfile. First two widths (0.031 & 3.5) are located at left side of alignment.
I will check vertical offsets.